### ISSUE TITLE
[Accessibility]A11y_PowershellEditorServices_Installation_Keyboard:Keyboard focus is not visible after the Tab items and screenreader reads some extra info that "Previous match button,next match button and close button".

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61712]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10
NVDA 2018.2
**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Navigate through keyboard such as "Tab" key.
4.Observed that whether the keyboard focus is visible after the Dependencies tab item and what narrator is reading.

**Actual:**
Keyboard focus is lost after the "Dependencies" tab item and narrator is reading Previous match button,next match button and close button after the dependencies button the buttons are not visible to the user.
**Same issue is observed in NVDA also.**
Expected:
Keyboard focus should visible that where is the focus after the dependencies tab item and also narrator should not read as extra content that  Previous match button,next match button and close button after the dependencies tabwhich are not visible to the user.
Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User will not able to know the where is the content present.
**MAS Reference:**
MAS2.4.3,MAS1.3.2
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}](url)
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference**
[2730781_A11y_PowershellEditorServices_Installation_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2510413/2730781_A11y_PowershellEditorServices_Installation_Keyboard.pptx)

**Comment by isidorn** [comment]
Try with latest vscode insiders, this should be fixed
